dwitter/templates/base.html

```diff
@@ -37,23 +37,26 @@
       // Check for compatibility
       if (window.localStorage){
         var switchTheme = function(){
-          // Change the imported stylesheet from main.css to main_dark.css or vice versa
-          // Change localstorage theme to 'dark'
+          // Shift themes
           var theme = localStorage.getItem('theme');
-          if (theme == 'dark') {
-            localStorage.setItem('theme','');
-          } else {
+          if (theme !== '') { 
             localStorage.setItem('theme','dark');
+          } else if (theme === 'dark') {
+            localStorage.setItem('theme','green');
+          } else {
+            localStorage.setItem('theme','');
           }
           location.reload();
           // Call this function in the li to change the mode
         }
 
         // Do the same thing outside the function without changing localStorage to set correct theme
         var theme = localStorage.getItem('theme');
-        if (theme == 'dark'){
+        if (theme !== '') {
```

Comment:
This is fine. But you could also write `if (theme) {`

Idiomatic Javascript is lazy!